lab01/lab01.py

```diff
@@ -68,4 +90,9 @@ def double_eights(n):
     False
     """
     "*** YOUR CODE HERE ***"
+    while n>10:
+        if n%10==8 and n%100==8 or n==88:
```

Comment:
Incorrect answer:

INFO:root:Calling function 'double_eights' with arguments [81881223] (call 3)
False

Even if the original assignment provide basic example input and output, better to test it comprehensively via various inputs.

Reply:
make correction based on given solution

Reply:
INFO:root:Calling function 'double_eights' with arguments [80808080] (call 1)
True

Still wrong answer. Make sure the correction not only work for any special case, but also general purpose of the function should persist.